crates/bin/pindexer/src/stake/validator_set.rs

```diff
@@ -28,7 +28,7 @@ impl AppView for ValidatorSet {
             // hence TEXT fields
             "CREATE TABLE stake_validator_set (
                 id SERIAL PRIMARY KEY,
-                ik BYTEA NOT NULL,
+                ik TEXT NOT NULL,
```

Comment:
Do we have to change this?

Reply:
Yes, the developer experience of using byte identity keys turns out to suck, so I fixed it